app/graphql/types/union/embed_item.rb

```diff
@@ -0,0 +1,5 @@
+class Types::Union::EmbedItem < Types::Union::Base
```

Comment:
Is there a good argument to using a union vs just referencing the base embed interface that all embeds inherit from?

Reply:
I'm not actually sure. I've always preferred using a union vs referencing the base embed, I think it reads nicer. But both function the same way.

Reply:
After reading this: https://artsy.github.io/blog/2019/01/14/graphql-union-vs-interface/ I think I prefer Unions.